### DISCUSSION TITLE
How to I create a custom style to make the labels bold?

### DISCUSSION THREAD

**Discussion by mapembert** [IBM/carbon-ui-builder#232] [Q&A] [answered]
Hello, I've been experimenting with styles but I can't seem to get a handle on the font-weight of the labels, for example.

Simple changes like "height: 10px;" work great for adding spaces around the grid however how do I target all Labels or a specific Label for a dropdown or something like it?

Thank you!

**Comment by zvonimirfras** [comment, marked as answer]
Hi Mike!

To add styling to all the components in a fragment, you can apply a class directly to the fragment in the same "Advanced styling" section, while no component is selected.

![image](https://user-images.githubusercontent.com/9692126/225972413-13334711-d7a8-4125-acbb-89c0c8678c05.png)

The content of the style class is SCSS, which mean you can nest any additional selectors in it. Then it's a matter of targeting a specific-enough selector.

For a label that would be

```css
.bx--label {
    font-weight: bold;
}
```

![image](https://user-images.githubusercontent.com/9692126/225972622-c8ee2c20-e123-4bbb-b6c7-2a88afacd86c.png)

I hope this helps. Feel free to reopen the discussion if there's anything else you'd like to know 😃
